features/flags.json

```diff
@@ -26,35 +26,75 @@
       "enabled": true,
       "lastEditedBy": "Andy Haynes",
       "lastEditedAt": "2022-01-31T21:13:09.948Z"
+    },
+    "mainnet_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:06:54.374Z"
+    },
+    "mainnet_STAGING_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:07:28.665Z"
+    },
+    "testnet_NEARORG": {
+      "enabled": true,
+      "lastEditedBy": "esaminu",
+      "lastEditedAt": "2022-05-05T18:07:50.110Z"
+    },
+    "testnet_STAGING_NEARORG": {
```

Comment:
`testnet_STAGING` doesn't really need to be duplicated since it won't be supported by any sites under `*wallet.near.org`. It really only exists now so that I can deploy to that environment with the new CI/CD pipeline, but since the domain migration takes priority I'm not going to bother deploying under the `*.near.org` domain. It's safe to ignore as part of this work 🙂

Reply:
deleted the environment 👍